app/src/test/java/com/codesoom/assignment/application/TaskServiceTest.java

```diff
@@ -0,0 +1,170 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.TaskNotFoundException;
+import com.codesoom.assignment.models.Task;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@DisplayName("TaskService 클래스")
+class TaskServiceTest {
+    private TaskService taskService;
+    private static final String TASK_TITLE = "test";
+    private static final Long NOT_EXIST_ID = 999999999L;
+
+    @BeforeEach
+    void setUp() {
+        taskService = new TaskService();
+    }
+
+    @Nested
+    @DisplayName("getTasks 메소드는")
+    class Describe_getTasks {
+        @Nested
+        @DisplayName("등록된 할 일이 없다면")
+        class Context_no_have_tasks {
+            @Test
+            @DisplayName("비어있는 리스트를 리턴한다")
+            void it_returns_emptyTask() {
+                List<Task> tasks = taskService.getTasks();
+                assertThat(tasks).isEmpty();
+            }
+        }
+
+        @Nested
+        @DisplayName("등록된 할 일이 있다면")
+        class Context_have_tasks {
+            @Test
+            @DisplayName("할 일들을 리턴한다")
+            void it_returns_tasks() {
+                Task task1 = new Task();
+                Task task2 = new Task();
+
+                taskService.createTask(task1);
+                taskService.createTask(task2);
+
+                // when
+                List<Task> tasks = taskService.getTasks();
+
+                //then
+                assertThat(tasks.size()).isEqualTo(2);
+                assertThat(tasks).contains(task1, task2);
+            }
+        }
+    }
+
+    @Nested
+    @DisplayName("getTask 메소드는")
+    class Describe_getTask {
+        @Nested
+        @DisplayName("입력받은 id와 일치하는 등록된 할 일이 없다면")
+        class Context_matchId_NotExist {
+            @Test
+            @DisplayName("할 일을 찾을 수 없다는 에러를 발생시킨다.")
```

Comment:
1. 예외와 에러는 다릅니다.
2. 내가 작성한 코드에서는 예외는 발생하는 것이 아니에요. 예외는 개발자가 "던지는" 것입니다. 우리가 `throw` 키워드를 키보드를 써서 타이핑한다는 사실을 떠올려 보세요.

```suggestion
            @DisplayName("할 일을 찾을 수 없다는 예외를 던진다.")
```